workspaces/tests/test-contracts/ft/src/lib.rs

```diff
@@ -0,0 +1,75 @@
+use near_contract_standards::fungible_token::metadata::{
+    FungibleTokenMetadata, FungibleTokenMetadataProvider, FT_METADATA_SPEC,
+};
+use near_contract_standards::fungible_token::FungibleToken;
+use near_sdk::borsh::{self, BorshDeserialize, BorshSerialize};
+use near_sdk::collections::LazyOption;
+use near_sdk::json_types::U128;
+use near_sdk::{
+    env, log, near_bindgen, require, AccountId, Balance, BorshStorageKey, PanicOnDefault,
+    PromiseOrValue,
+};
+
+#[near_bindgen]
+#[derive(BorshDeserialize, BorshSerialize, PanicOnDefault)]
+pub struct Contract {
+    token: FungibleToken,
+    metadata: LazyOption<FungibleTokenMetadata>,
+}
```

Comment:
Why is FT contract used specifically? Seems like you're just trying to make sure that it can be compiled and called in a test? Seems like having a more involved test like this would just be more annoying for maintenance unless there is a reason for it?

Reply:
Good point, replaced with `status-message` which is much simpler.